templates_library/async/repository_implementation.mustache

```diff
@@ -0,0 +1,21 @@
+//
+//  {{name}}RepositoryImplementation.swift
+//  {{project_name}}
+//
+//  Created by {{full_username}} on {{date}}.
+//
+//
+//
```

Comment:
This file is the only one to have 3 empty lines in the header, we should stick to 2.

And we should also fix it in `lib/ccios/templates/repository_implementation.mustache`